lib/layout.js

```diff
@@ -561,8 +555,9 @@ export default class PodiumLayout {
         incoming.css = [...this.cssRoute];
         incoming.js = [...this.jsRoute];
 
-        // @ts-ignore
+        // @ts-expect-error Context hasn't been updated yet with the new typings for HttpIncoming.
         if (context) await this.context.process(incoming);
+        // @ts-expect-error Proxy hasn't been updated yet with the new typings for HttpIncoming.
```

Comment:
TODO: `@podium/proxy` types from JSDoc, although this one in particular might be handled by renovate-bot